client/payment-details/dispute-details/dispute-awaiting-response-details.tsx

```diff
@@ -346,7 +358,7 @@ const DisputeAwaitingResponseDetails: React.FC< Props > = ( {
 															'transaction_details',
 													}
 												);
-												setModalOpen( false );
```

Comment:
Curious why we didn't use `onModalClose` before.

Reply:
Same!